kie-server-parent/kie-server-services/kie-server-services-openshift/src/main/java/org/kie/server/services/openshift/api/KieServerOpenShiftUtils.java

```diff
@@ -15,20 +15,29 @@
 
 package org.kie.server.services.openshift.api;
 
+import java.util.Optional;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
 import io.fabric8.openshift.api.model.DeploymentConfig;
+import io.fabric8.openshift.client.OpenShiftClient;
 import org.kie.server.services.impl.KieServerLocator;
 
-public interface KieServerReadinessProbe {
+public interface KieServerOpenShiftUtils {
```

Comment:
why is this an interface with default methods instead of regular class with static methods?

Reply:
Main reason is to facilitate test. (https://github.com/rhtevan/droolsjbpm-integration/blob/ad146251274aa777f3f25dae89225c7d24049a6e/kie-server-parent/kie-server-services/kie-server-services-openshift/src/test/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepositoryTest.java#L90-L116)

Reply:
alright, fair enough ... though the name of the interface sounds bit odd...

Reply:
I know. How about just name that interface as 'KieServerOpenShift'?

Reply:
+1 I find it much better than ... Utils as interface

Reply:
Changed. don't have anything more before merge from my side. Do you have further suggestions?

Reply:
nope, once build is done I'll merge it